src/JuMP.jl

```diff
@@ -216,6 +218,15 @@ for f in MathProgBase.SolverInterface.methods_by_tag[:rewrap]
     eval(Expr(:export,f))
 end
 
+function getobjectivebound(m::Model)
+    # check internal model exists
+    if !m.internalModelLoaded
```

Comment:
This check isn't needed. Just implement a method consistent with ``getobjectivevalue``.